test/simplify_dyn_ops_test.cpp

```diff
@@ -237,4 +237,32 @@ TEST_CASE(const_slice_4input)
     EXPECT(m0 == m1);
 }
 
+TEST_CASE(static_dimensions_of)
+{
+    migraphx::module m0;
+    {
+        migraphx::shape s{migraphx::shape::float_type, {2, 4, 4}};
+        m0.add_parameter("data", s);
+        migraphx::shape lit_shape{migraphx::shape::int64_type, {3}};
+        ;
+        std::vector<int64_t> lit_data = {2, 4, 4};
+        auto lit_ins                  = m0.add_literal(migraphx::literal{lit_shape, lit_data});
+        m0.add_return({lit_ins});
+    }
+
+    // dead_code_elimination will get rid of atan
+    migraphx::module m1;
```

Comment:
Can put this first? In the other tests the input module is first and the expected output is last.